packages/grafana-ui/src/components/ConfirmButton/ConfirmButton.mdx

```diff
@@ -0,0 +1,11 @@
+<Meta title="MDX|ConfirmButton" component={ConfirmButton} />
+
+# ConfirmButton
+
+The ConfirmButton is an interactive component that adds a double-confirm option to a clickable action. When clicked, the action is replaced by an inline confirmation with the option to cancel. In Grafana, this is used for example for editing values in settings tables.
```

Comment:
`double-confirm option`? This sounds odd :D 

`The ConfirmButton is an interactive component that provides additional security for actions being performed by asking the user to confirm the action` maybe?